src/api/userManagementApi.ts

```
@@ -216,113 +221,185 @@ export const createUser = async (userData: User): Promise<User> => {
+      formattedData.profession = userData.profession || '';
+      formattedData.contact = userData.parentContact || ''; // Change parentContact to contact here too
       break;
-  }
 
-  // Remove any undefined or null values from the nested data objects
-  if (formattedData.teacherData) {
-    Object.keys(formattedData.teacherData).forEach(key => {
-      if (formattedData.teacherData[key] === undefined || formattedData.teacherData[key] === null || formattedData.teacherData[key] === '') {
-        delete formattedData.teacherData[key];
+    case "Student":
+      const studentData: Record<string, any> = {};
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)